src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -2405,6 +2435,26 @@ public class Person
             public int Id { get; set; }
         }
 
+        public class DataQueryOptions
+        {
+            [FromQuery(Name = "paging")]
+            public DataQueryPaging Paging { get; set; }
```

Comment:
It would be much more useful to test these scenarios:

```C#
public class Model1
{
    public string Property1 { get; set; }
}

public IActionResult Action1([FromQuery(Name = "model1")] Model1 model)
```

and 

```C#
public class Model2
{
   [FromQuery(Name = "property1")] public string Property1 { get; set; }
   
   [FromRoute(Name = "property2")] public string Property2 { get; set; }
}

public IActionResult Action1([ModelBinder] Model2 model)
```

We don't need to test complicated hierarchies.

Reply:
ok